### PR TITLE
Turn echo off when reading passwords

### PR DESCRIPTION
**Pull request by tuxcanfly**
Used the cross platform util  `ReadPassword` provided by `code.google.com/p/go.crypto/ssh/terminal` for this.

### CODE REVIEW

**Comment by davecgh** [comment]
I haven't tested it yet, but it reads well.  Thanks!

File: walletsetup.go [hunk 1]
```diff
@@ -81,17 +84,18 @@ func promptConsoleListBool(reader *bufio.Reader, prefix string, defaultEntry str
...
 // The function will ask the user to confirm the passphrase and will repeat
 // the prompts until they enter a matching response.
-func promptConsolePass(reader *bufio.Reader, prefix string, confirm bool) (string, error) {
+func promptConsolePass(reader *bufio.Reader, prefix string, confirm bool) ([]byte, error) {
 	// Prompt the user until they enter a passphrase.
 	prompt := fmt.Sprintf("%s: ", prefix)
 	for {
 		fmt.Print(prompt)
-		pass, err := reader.ReadString('\n')
+		pass, err := terminal.ReadPassword(int(os.Stdout.Fd()))
```
Comment (davecgh) [inline walletsetup.go:92]: I believe this should be os.Stdin.Fd().

File: walletsetup.go [hunk 2]
```diff
@@ -100,12 +104,13 @@ func promptConsolePass(reader *bufio.Reader, prefix string, confirm bool) (strin
 		}
 
 		fmt.Print("Confirm passphrase: ")
-		confirm, err := reader.ReadString('\n')
+		confirm, err := terminal.ReadPassword(int(os.Stdout.Fd()))
```
Comment (davecgh) [inline walletsetup.go:107]: Same here.

**Comment by davecgh** [comment]
Alright, I confirmed those lines should indeed by `os.Stdin.Fd()`.  Otherwise you (rightfully) get  an invalid handle error.  With those changes, everything works as expected.  I've made those changes, squashed, and rebased on top of the `davec_intwaddrmgr` branch, and merged it as commit a83cfefaeb81407f32d0a946ef4e6d12076cfcfc.

**Comment by tuxcanfly** [comment]
Thanks @davecgh it should have been `os.Stdin.Fd()` indeed.

**Pull request closed without merging**